test/FsAutoComplete.Tests.Lsp/Utils/Server.Tests.fs

```diff
@@ -276,7 +276,6 @@ let tests state = testList (nameof(Server)) [
 
     testSequenced <| testList "contesting" [
       let projectDir = inTestCases "Project"
-      dotnetRestore projectDir.Value |> Async.RunSynchronously
```

Comment:
This was causing using the new debug feature to fail. Guessing this was here before we started doing restores in the `serverInit` functions.